addons/website_event/models/event_event.py

```diff
@@ -531,10 +531,20 @@ def _search_get_detail(self, website, order, options):
             mapping['description'] = {'name': 'subtitle', 'type': 'text', 'match': True}
         if with_date:
             mapping['detail'] = {'name': 'range', 'type': 'html'}
+
+        def search_in_city_or_country(env, search_term):
+            events_ids = env['event.event'].sudo().search([
+                '|',
+                ('address_id.city', 'ilike', search_term),
+                ('address_id.country_id', 'ilike', search_term)
```

Comment:
You should put a comma `,` at the end of the last entry of the domain array, to help minimize the impacted lines in future commits.